src/index.js

```diff
@@ -0,0 +1,204 @@
+const state = {
+  temp: {
+    cTemp: 20,
+    fTemp: 68,
+  },
+  sky: 'sunny',
+  cityName: 'Tacoma',
+  tempType: 'f',
+};
+
+// rendering
+const renderTemp = () => {
+  const {
+    temp: { cTemp, fTemp },
+    tempType,
+  } = state;
+  const tempElem = document.querySelector('#current-temp');
+  tempElem.className = getTempClass();
+
+  tempElem.textContent = `${tempType === 'f' ? `${fTemp}°F` : `${cTemp}°C`}`;
+};
+
+const renderLandscape = () => {
+  const landscapeElem = document.querySelector('#landscape');
+  landscapeElem.textContent = getLandscapeContent();
+};
+
+const renderSky = () => {
+  const skyElem = document.querySelector('#sky');
+  skyElem.textContent = getSkyContent();
+};
+
+const renderCityName = () => {
+  const { cityName } = state;
+  const nameElem = document.querySelector('#city-name');
+  nameElem.textContent = cityName;
+};
+
+const renderConvertBtn = () => {
+  const { tempType } = state;
+  const convertBtn = document.querySelector('#convert-btn');
+  convertBtn.textContent = tempType === 'f' ? 'To Fahrenheit' : 'To Celsius';
+};
+
+const renderAll = () => {
+  renderTemp();
+  renderLandscape();
+  renderSky();
+  renderCityName();
+  renderConvertBtn();
+};
+
+// helper logic for rendering
+const getTempClass = () => {
```

Comment:
Notice that `getTempClass` and `getLandscapeContent` have the same conditional tests. Consider refactoring and combining to DRY up your code.